contracts/bridges/succinct/SuccinctBridgeReceiver.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "../BaseBridgeReceiver.sol";
+import "./ITelepathy.sol";
+
+contract SuccinctBridgeReceiver is BaseBridgeReceiver, ITelepathyHandler {
+    ITelepathyReceiver public telepathyReceiver;
+
+    constructor(address _telepathyReceiver) {
+        telepathyReceiver = ITelepathyReceiver(_telepathyReceiver);
+    }
+
+    function handleTelepathy(uint32 _sourceChainId, address _sourceAddress, bytes calldata _data) external returns (bytes4)  {
+        processMessage(_sourceAddress, _data);
+        return ITelepathyHandler.handleTelepathy.selector;
+    }
+
+    receive() external payable {
+    }
```

Comment:
Is there a reason this contract needs a `receive()`?

Reply:
No, it doesn't, will remove.